src/instrumentation/podmutator.go

```diff
@@ -35,150 +42,244 @@ var (
 )
 
 type instPodMutator struct {
-	Client      client.Client
-	sdkInjector *sdkInjector
-	Logger      logr.Logger
-}
-
-type languageInstrumentations struct {
-	Java   *v1alpha1.Instrumentation
-	NodeJS *v1alpha1.Instrumentation
-	Python *v1alpha1.Instrumentation
-	DotNet *v1alpha1.Instrumentation
-	Php    *v1alpha1.Instrumentation
-	Ruby   *v1alpha1.Instrumentation
-	Go     *v1alpha1.Instrumentation
+	client                 client.Client
+	sdkInjector            SdkInjector
+	secretReplicator       SecretReplicator
+	instrumentationLocator InstrumentationLocator
+	logger                 logr.Logger
+	operatorNamespace      string
 }
```

Comment:
nit: let's define these struct fields in the order we are calling these in the args in the NewMutator function for clarity.

Reply:
fixed